tensorflow/lite/examples/python/label_image.py

```diff
@@ -100,4 +100,5 @@ def load_labels(filename):
     else:
       print('{:08.6f}: {}'.format(float(results[i] / 255.0), labels[i]))
 
-  print("time: ", stop_time - start_time)
+  #print("time: ", stop_time - start_time)
```

Comment:
please remove :)

Reply:
oops, that's embarrassing. Thanks.

Reply:
No worries, it happens all the time for me too ;)